ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1384,6 +1384,16 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-oidc-private-f14
```

Comment:
Suggest to add the job into `openshift-openshift-tests-private-release-4.16__multi-nightly.yaml`, also sounds missing a destructive ci job.

Reply:
Thank you. I will add destructive ci job and move to `openshift-openshift-tests-private-release-4.16__multi-nightly.yaml`